src/store/indexedDB.js

```diff
@@ -16,6 +16,12 @@ class IndexedDB {
     );
   }
 
+  markDecryptionFailedMessages(mids) {
```

Comment:
So we have 2 functions now to update statuses:

```
  markMessagesAsRead(mids) {
    this.db.messages.bulkUpdate(
      mids.map((id) => ({ key: id, changes: { status: "read" } }))
    );
  }

  markDecryptionFailedMessages(mids) {
    this.db.messages.bulkUpdate(
      mids.map((id) => ({ key: id, changes: { status: "decryption_failed" } }))
    );
  }
```

I propose to universalize it:

```
  updateMessagesStatus(mids, status) {
    this.db.messages.bulkUpdate(
      mids.map((id) => ({ key: id, changes: { status } }))
    );
  }
```

Reply:
I had the same idea), but I decided that every time the method is called, writing the status would look not good

Also fixed this comment for Messages redux